src/Nethermind/Nethermind.JsonRpc/Data/AccessListItemForRpc.cs

```diff
@@ -8,12 +8,15 @@
 using Nethermind.Core.Eip2930;
 using Nethermind.Int256;
 using Nethermind.Serialization.Json;
-using Newtonsoft.Json;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 
 namespace Nethermind.JsonRpc.Data
 {
     public class AccessListItemForRpc
     {
+        [JsonConstructor]
+        public AccessListItemForRpc() { }
```

Comment:
can it be private?

Reply:
In .NET 7 and earlier versions, the [JsonConstructor] attribute can only be used with public constructors.